src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -77,7 +77,7 @@
     // subject to future changes as we see more test cases
     @VisibleForTesting static final int MIN_OVERLAP_IN_FILTER_OVERLAP_ASSEMBLE = 55;
     @VisibleForTesting static final int FILTER_STEP_KMER_FREQUENCY_THREASHOLD = 1;
-    @VisibleForTesting static final double OVERLAP_STEP_ERROR_RATE = 0.05;
+    @VisibleForTesting static final double OVERLAP_STEP_ERROR_RATE = 0.02;
```

Comment:
While we're changing parameters we might as well turn these into optional command line arguments with these as defaults.